app/views/multifactor_auths/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<% @title = t('.title') %>
+
+<div class="t-body l-overflow">
+  <div class="l-half--l">
+    <%= raw @qrcode_svg %>
+  </div>
+  <div>
+    <p><%= t('.scan_prompt') %></p>
+    <p>
+      <span id="mfa-account"><%= t('.account', :account => "#{issuer}:#{current_user.email}") %></span><br>
+      <span id="mfa-key"><%= t('.key', :key => @seed.chars.each_slice(4).map(&:join).join(' ')) %></span><br>
+      <span id="mfa-type"><%= t('.time_based') %></span>
```

Comment:
I don't think there is any value in showing `account` and `time_based` here. I haven't seen it any other website.